CO2_Cleanup.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.                data.dropna(subset=['datetime'], inplace=True)  # Drop rows with invalid datetime
</pre><p>This will lead to uneven spaced data that cannot be used in this FFT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1-24-HW5/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>